cosmos/azure-spring-data-cosmos/cosmos-multi-tenant-samples/tenant-by-container/src/main/java/com/azure/spring/data/cosmos/example/UserController.java

```diff
@@ -0,0 +1,65 @@
+package com.azure.spring.data.cosmos.example;
+import com.azure.cosmos.models.PartitionKey;
+import com.azure.spring.data.cosmos.example.tenant.TenantStorage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Controller;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+@PropertySource("classpath:application.yaml")
+@Controller
+@ContextConfiguration(classes = AppConfiguration.class)
```

Comment:
Remove this line, it is only for testing purposes.

Reply:
ok, thanks